### PR TITLE
Better Promise usage

### PR DESCRIPTION
**Pull request by TimothyGu**

### CODE REVIEW

**Comment by tunnckoCore** [comment]
:+1:

File: list.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict'
 
-var github = require('github-repositories')
 var Promise = require('promise')
+var github = Promise.denodeify(require('github-repositories'))
```
Comment (RobLoach) [inline list.js:4]: I like where you're going with this, but [`Promise.denodeify`](https://www.npmjs.com/package/promise#promise-denodeify-fn) isn't quite considered standard ES6. Could we use something that works with straight up Promises?

Comment (TimothyGu) [inline list.js:4]: I understand, but Node.js 0.10 isn't going away any time soon, and the jstransformer core is already using the denodeify function anyway. Plus this module isn't supposed to be Browserified either so we don't really have a reason not to use the extensions provided by the module.

Comment (RobLoach) [inline list.js:4]: Sounds good with me!

**Pull request merged**